src/graphql/mutations/createEtchPacket.js

```diff
@@ -8,19 +8,21 @@ const defaultResponseQuery = `{
     eid
     status
     files
+    providerConfig
```

Comment:
@benogle Do we want `providerConfig` to be directly referenced/accessed by our Users? Or should we expose the specific things from `providerConfig` that they might need as fields on this object?

Reply:
What I can do as well is remove providerConfig from the defaultResponseQuery, and just add my custom responseQuery with providerConfig in it.

Reply:
Ya, maybe that's the move...but still you will be accessing a deep, dark, private place in a public _**example**_ repo.

If @benogle 's ok with it and/or I'm mistaken about the stance of not wanting users to be directly accessing the `providerConfig`, then go right ahead...but AFAIK this `providerConfig` is "off limits" to the public...and if there's something they need from it, we should abstract it with some sort of accessor/field in GraphQL

Reply:
Let me talk with Ben about it

Reply:
@newhouse I've now removed providerConfig from the query and instead added in `detailsURL` and `downloadZipURL` to get the fields I need.